docs/contributing/muzzle.md

```diff
@@ -30,6 +30,11 @@ the `InstrumentationModule#isHelperClass(String)` predicate). Aside from referen
 the collection process also builds a graph of dependencies between internal instrumentation helper
 classes - this dependency graph is later used to construct a list of helper classes that will be
 injected to the application classloader (`InstrumentationModule#getMuzzleHelperClassNames()`).
+Muzzle also automatically generates the `InstrumentationModule#getMuzzleContextStoreClasses()`
+method.
+
+If you extend any of these `getMuzzle...()` methods in your `InstrumentationModule`, the muzzle
+compile plugin will not override your code: muzzle will skip those custom methods.
```

Comment:
This sounds like muzzle with skip MY custom methods.

Reply:
Reworded